nginx.proxy.conf.esh

```diff
@@ -0,0 +1,90 @@
+<%# We have to set the default values for variables here because
+    AWK throws 'unbound variable' errors otherwise -%>
```

Comment:
I wouldn't mention "have to" and "AWK  throws ..." because we actually want explicit defaults to be set IMO to match the description in the README without reading the shell script.